src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -139,4 +142,4 @@ <h6 class="is-6 is-small-font">
         [tabs]="groupTabs" (selectedTabEvent)="selectedGroup = $event"></app-tab-navigation>
```

Comment:
Here,
`selectedGroup = $event` should be changed to `selectedGroup.key = $event` or not?

Reply:
No, selectedGroup holds an object of {key, value} and on tab switch we need to update the group not only key of the group.